lib/setup/setupList.js

```diff
@@ -414,26 +414,29 @@ function List(options) {
                                     name += new Array(22 - name.length).join(' ');
                                 }
 
-                                let text = id + ': ' + (name || '') + ' - ' +
-                                    (objs.rows[i].value.common.enabled ? ' enabled' : 'disabled');
+                                if (host && host.length < 40) {
+                                    host += new Array(40 - host.length).join(' ');
+                                }
```

Comment:
were you looking for "string".padStart?

Reply:
Used the same stuff as above, see https://github.com/ioBroker/ioBroker.js-controller/pull/931/files#diff-6672f1152a1a12f37c77ffbc89e0b4a6R413 but `padStart` sounds good and more self-explaining.